### PR TITLE
Added new VueJS app

### PR DESCRIPTION
**Pull request by MrToino**
Implemented a simple VueJS app to allow file updating

### CODE REVIEW

File: services/test_file/test.csv [hunk 1]
```diff
@@ -0,0 +1,5 @@
+x,y1,y2
```
Comment (luisferreira32) [inline services/test_file/test.csv:1]: weren't test files placed somewhere else, like in test/ folder?

**Review by luisferreira32 — APPROVED** [review]
I'd say you don't need to add `LoG` prefix (I know it means Lion of Graphics), to every component file. We're in the repo itself... but the rest seems fine :+1:

File: services/test_file/test.csv (hunk 1 above)
Comment (MrToino) [reply services/test_file/test.csv:1]: > I'd say you don't need to add `LoG` prefix (I know it means Lion of Graphics), to every component file. We're in the repo itself... but the rest seems fine 👍

I had to put some name as some components such as Headers must be multi-worded (vue demands that), as such I set them with LoG, but that can be changed later on.

Regarding that test file, yes that was a mistake and I'll correct right away.

**Comment by luisferreira32** [comment]
I've got to fix the pipelines, they only run at first and when you edit the PR, but not actually when you push new changes... fixed it for you, and sorry about that :)

**Pull request merged**